Bottom: 412a65555052c1d9c94a4573b2f2a044b74ad7ec
Top:    525bd0530a91934c272824ed617bdad81f6aab89
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-10 22:52:40 +0100

Make --enable-dbus default


---

diff --git a/configure.ac b/configure.ac
index 08ffa53..863e2ee 100644
--- a/configure.ac
+++ b/configure.ac
@@ -353,7 +353,13 @@ fi
 AC_MSG_CHECKING([for whether to use DBUS])
 AC_ARG_ENABLE([dbus],
 [  --enable-dbus           Enable DBUS IPC],
-[],[enable_dbus=no])
+[],[
+	case " $with_gui " in
+		*\ gtk\ *)     enable_dbus=yes ;;
+		*\ lesstif\ *) enable_dbus=yes ;;
+		* )            enable_dbus=no ;;
+	esac
+])
 
 AC_MSG_RESULT([$enable_dbus])
 AM_CONDITIONAL(WITH_DBUS, test x$enable_dbus = xyes)
